uaa/src/test/java/org/cloudfoundry/identity/uaa/orchestrator/UaaOrchestratorClaimTest.java

```diff
@@ -0,0 +1,301 @@
+package org.cloudfoundry.identity.uaa.orchestrator;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.ge.predix.iam.kubernetes.constant.Constants;
+import com.ge.predix.iam.kubernetes.enums.ApiEnvironment;
+import com.ge.predix.iam.kubernetes.enums.ServiceName;
+import com.ge.predix.iam.kubernetes.exception.ServiceInstanceProviderException;
+import com.ge.predix.iam.kubernetes.model.Claim;
+import com.ge.predix.iam.kubernetes.model.uaa.UaaClaimSpec;
+import com.ge.predix.iam.kubernetes.model.uaa.UaaForProvider;
+import com.ge.predix.iam.kubernetes.model.uaa.UaaRequest;
+import com.ge.predix.iam.kubernetes.utils.OrchestratorUtil;
+import io.kubernetes.client.openapi.ApiException;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang.StringEscapeUtils;
+import org.apache.commons.lang.StringUtils;
+import org.apache.http.config.SocketConfig;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.cloudfoundry.identity.uaa.zone.model.OrchestratorZoneResponse;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.*;
+import org.springframework.http.client.ClientHttpRequestFactory;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+import org.springframework.security.oauth2.client.DefaultOAuth2ClientContext;
+import org.springframework.security.oauth2.client.OAuth2RestOperations;
+import org.springframework.security.oauth2.client.OAuth2RestTemplate;
+import org.springframework.security.oauth2.client.resource.OAuth2ProtectedResourceDetails;
+import org.springframework.security.oauth2.client.token.DefaultAccessTokenRequest;
+import org.springframework.security.oauth2.client.token.grant.client.ClientCredentialsResourceDetails;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.util.Random;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static org.junit.Assert.*;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = OrchestratorIntegrationTestConfig.class)
+@Slf4j
+public class UaaOrchestratorClaimTest {
```

Comment:
Did you configure this test UaaOrchestratorClaimTest anywhere in build.gradle or jenkins file? Just want to know because it should run as part of Acceptenace Tests EKS jenkins pipeline https://i.ci.build.ge.com/bjmjek1k/ci/job/UAA/job/Acceptance%20Test%20EKS/

Reply:
Done